starter-workflows/code-scanning/frogbot-scan-and-fix.yml

```diff
@@ -40,21 +39,26 @@ jobs:
       #    with:
       #      node-version: "16.x"
 
+      # The full template list with the required GitHub Actions can be found at https://github.com/jfrog/frogbot/tree/master/docs/templates/github-actions/scan-and-fix
 
-      - uses: jfrog/frogbot@9304d3b1d8e05a1b5fc0ba9ebf9ffbd495386250
+      - uses: jfrog/frogbot@80d5c273e67898731f4d20cb5eb218f9f15b3afd
         env:
           # [Mandatory]
           # JFrog platform URL (This functionality requires version 3.29.0 or above of Xray)
```

Comment:
Let's remove this part from the description - 
```
(This functionality requires version 3.29.0 or above of Xray)
```
There's probably no need for this section to be included only for the JF_URL parameter.